hybridse/src/udf/default_udf_library.cc

```diff
@@ -1609,7 +1609,28 @@ void DefaultUdfLibrary::InitTimeAndDateUdf() {
                 select dayofweek(timestamp(1590115420000));
                 -- output 6
             @endcode
-            @since 0.1.0
+            @since 0.4.0
```

Comment:
revert this line

Reply:
`dayofweek()` was introduced into OpenMLDB since 0.1.0 @Nicholas-SR